lib/gui/app/app.js

```diff
@@ -410,6 +410,59 @@ app.run(($state) => {
   })
 })
 
+// Let there be light!
+if (/^(1|true)$/i.test(process.env.ETCHER_BLINKENLIGHTS)) {
+  app.run(() => {
+    const BlinkenLights = require('../../shared/blinkenlights')
+    const lights = new BlinkenLights({
+      i2cBus: parseInt(process.env.LED_I2C_BUS, 10) || null,
+      address: parseInt(process.env.LED_CONTROLLER_ADDRESS, 10) || null,
+      frequency: parseInt(process.env.LED_CONTROLLER_FREQUENCY, 10) || null,
+      debug: /^(1|true)$/i.test(process.env.LED_CONTROLLER_DEBUG),
+    })
+
+    lights.init(null, (error) => {
+      if (error) {
+        console.error(error)
+        return
+      }
+
+      // NOTE(jhermsmeier): I have no idea what this might do;
+      // it's straight from the https://github.com/101100/pca9685
+      // example in the README.md
+
+      // Set channel 0 to turn on on step 42 and off on step 255
+      // (with optional callback)
+      pwm.setPulseRange(0, 42, 255, (error) => {
```

Comment:
Are you missing a `const pwm = lights.driver` or similar?

Reply:
Oh, yes indeed 🤦‍♂️